build/config/phone/custom-prefs.js

```diff
@@ -2,3 +2,4 @@
 user_pref('devtools.responsiveUI.customWidth', 320);
 user_pref('devtools.responsiveUI.customHeight', 480);
 user_pref('devtools.responsiveUI.currentPreset', 'custom');
+user_pref('browser.sessionhistory.max_total_viewers', 3);
```

Comment:
I'll remove it before asking for the final review. Im keeping it right now just to test the result properly.